Geometry/HGCalCommonData/interface/HGCalGeomRotation.h

```diff
@@ -0,0 +1,15 @@
+#ifndef Geometry_HGCalCommonData_HGCalGeomRotation_h
+#define Geometry_HGCalCommonData_HGCalGeomRotation_h
+
+class HGCalGeomRotation {
```

Comment:
Is there a plan to add more to this class in the future? As it is, a namespace rather than a class would suffice, since there are no data members.

Reply:
It is intended that this class would grow in the future. I have also now added the option to rotate by 60º (for the CE-E), which has necessitated the addition of a data member.